crates/bcr-wdc-webapi/src/quotes.rs

```diff
@@ -151,6 +156,7 @@ pub struct ListReply {
 pub struct LightInfo {
     pub id: uuid::Uuid,
     pub status: StatusReplyDiscriminants,
+    pub sum: BitcoinAmount,
```

Comment:
Ok

I did some research, I can walk you through it later when pairing maybe

We don't need to wrap `bitcoin::Amount` around our own type, we can instruct `utoipa` to interpret this as a u64

You can change this line with
```suggestion
    #[schema(value_type = u64))]
    pub sum: bitcoin::Amount,
```